pages/linux/lvconvert.md

```diff
@@ -0,0 +1,24 @@
+# lvconvert
+
+> Convert or modify the type, redundancy, or state of logical volumes.
+> More information: <https://manned.org/lvconvert>.
+
+- Convert a linear logical volume to RAID1 (add one mirror: two copies total):
+
+`lvconvert --type raid1 {{[-m|--mirrors]}} {{1}} {{/dev/vg_name/lv_name}}`
```

Comment:
```suggestion
`lvconvert --type raid1 {{[-m|--mirrors]}} 1 {{/dev/vg_name/lv_name}}`
```
This is explicitly named in the description, thus the placeholder is not needed.